### PR TITLE
Use Exposed's JSONB support

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Please see the commit messages for details.

### CODE REVIEW

File: plugins/commands/upload-result-to-postgres/src/main/kotlin/UploadResultToPostgresCommand.kt [hunk 1]
```diff
@@ -145,5 +145,5 @@ class UploadResultToPostgresCommand : OrtCommand(
 }
 
 private class OrtResults(tableName: String, columnName: String) : IntIdTable(tableName) {
-    val result: Column<OrtResult> = jsonb(columnName, OrtResult::class)
+    val result: Column<OrtResult> = jsonb(columnName)
```
Comment (sschuberth) [inline plugins/commands/upload-result-to-postgres/src/main/kotlin/UploadResultToPostgresCommand.kt:148]: Nit: Should we write this as `val result = jsonb<OrtResult>(columnName)` now?

File: scanner/src/main/kotlin/storages/utils/JsonbSupport.kt [hunk 2]
```diff
@@ -19,6 +19,7 @@
 
 package org.ossreviewtoolkit.scanner.storages.utils
 
+import com.fasterxml.jackson.module.kotlin.readValue
```
Comment (sschuberth) [inline scanner/src/main/kotlin/storages/utils/JsonbSupport.kt:22]: Wrong import grouping.

File: scanner/src/main/kotlin/storages/utils/ScanResultDao.kt [hunk 3]
```diff
@@ -30,7 +30,7 @@ import org.ossreviewtoolkit.model.ScanResult
 
 object ScanResults : IntIdTable("scan_results") {
     val identifier: Column<String> = text("identifier").index("identifier")
-    val scanResult: Column<ScanResult> = jsonb("scan_result", ScanResult::class)
+    val scanResult: Column<ScanResult> = jsonb("scan_result")
```
Comment (sschuberth) [inline scanner/src/main/kotlin/storages/utils/ScanResultDao.kt:33]: See above. As a side note, line 32 could also omit the explicit type, if desired.

**Review by sschuberth — CHANGES_REQUESTED** [review]

**Review by sschuberth — APPROVED** [review]

**Comment by mnonnenmacher** [comment]
Merged as test failures are unrelated.

**Pull request merged**